App Service beta-3.0: missed docs and tweaks (#13322)
hashicorp/terraform-provider-azurerm: website/docs/r/linux_web_app.html.markdown

A `auth_settings` block supports the following:

> * `enabled` - (Required) Should the Authentication / Authorization feature is enabled for the Linux Web App?

Reviewer: ```suggestion
* `enabled` - (Required) Should the Authentication / Authorization feature be enabled for the Linux Web App?
```

A `identity` block supports the following:

> * `type` - (Required) The type of managed service identity. Possible values include: `SystemAssigned`, `UserAssigned`, and `SystemAssignedUserAssigned`.

Reviewer: the API defines these as:

```suggestion
* `type` - (Required) The type of managed service identity. Possible values include: `SystemAssigned`, `UserAssigned`, and `SystemAssigned, UserAssigned`.
```

> * `type` - (Required) The type of managed service identity. Possible values include: `ManagedServiceIdentityTypeSystemAssigned`, `ManagedServiceIdentityTypeUserAssigned`, and `ManagedServiceIdentityTypeSystemAssignedUserAssigned`.

Reviewer: ```suggestion
* `type` - (Required) The type of managed service identity. Possible values include: `SystemAssigned`, `UserAssigned`, and `SystemAssigned, UserAssigned`.
```
